media/libs/vue.min.js

```diff
@@ -1,6 +1,5018 @@
 /*!
- * Vue.js v2.5.16
- * (c) 2014-2018 Evan You
+ * Vue.js v2.6.12
```

Comment:
This was pulled in automatically by npm, I presume because a new version is available. I can restrict the vue version to `2.5.16` in the `package.json` if the toolkit team thinks that is appropriate.

Reply:
Can move forward with Vue update, though we should check that our webviews didn't regress:

- feedback
- Lambda remote invoke